### PR TITLE
Visualize Time Range

### PR DESCRIPTION
**Pull request by prit4321**
## Testing: npm run test:e2e

 ## Screenshots:
Bj0YvASP0y

### CODE REVIEW

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const { SpecReporter } = require("jasmine-spec-reporter");
 exports.config = {
   allScriptsTimeout: 11000,
   specs: ["./src/**/*e2e-spec.ts"],
-  // specs:[".src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts"],
+  specs:["./src/features/visualize/visualizeview/timeRange.e2e-spec.ts"],
```
Comment (FarajiA) [inline e2e/protractor.conf.js:9]: I think this should be account for all spec files.

File: e2e/src/pages/visualizeViewPage.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    startTimeMIN               =element(by.css("label.owl-dt-timer-content:nth-child(3) > input:nth-child(1)"));
+
+    getDate(date) {​​​​​​​​
+        console.log("Visualize", date);
+       
+        var d = new Date(date);
+          return [d.getFullYear(), `0${​​​​​​​​d.getMonth() + 1}​​​​​​​​`, d.getDate()].join('-');
+    }
+
+    // updateQueryStringParameter(){
```
Comment (FarajiA) [inline e2e/src/pages/visualizeViewPage.ts:46]: We can delete the commented out stuff?

**Review by FarajiA — APPROVED** [review]

File: e2e/protractor.conf.js (hunk 1 above)
Comment (FarajiA) [reply e2e/protractor.conf.js:9]: We also comment out line #8

Comment (prit4321) [reply e2e/protractor.conf.js]: i have updated

File: e2e/src/pages/visualizeViewPage.ts (hunk 2 above)
Comment (prit4321) [reply e2e/src/pages/visualizeViewPage.ts]: I have deleted

**Pull request merged**